app/assets/javascripts/live_search.js

```diff
@@ -109,15 +109,29 @@
     }
   };
 
+  LiveSearch.prototype.getAdditionalCustomDimensionsForPageView = function() {
+    var custom_dimensions = {};
+
+    // Custom dimension for most relevant result shown
+    var $results = $('.finder-results');
+    var $mostRelevantDocumentLink = $results.find('.document__link--top');
+    if ($mostRelevantDocumentLink.length) {
+      custom_dimensions["dimension83"] = { value: "yes" };
```

Comment:
Sorry I'm not familiar with how we're meant to set custom dimensions. Is this meant to be setting `dimension83` to `{value: "yes" }` or should it just be setting it to `"yes"`?

Reply:
You're correct, I already found the issue and pushed up 👍